### DISCUSSION TITLE
usb drive

### DISCUSSION THREAD

**Discussion by kosto-stavr** [rern/rAudio#143] [Fixed Issues]
I never used an usb drive to listen music till today, so I don’t know if it is a normal behavior.
The usb drive is auto mount and it is visible in Sources, but the contained tracks are not visible in library under usb.
Also,
By clicking the top left logo on players page, once upon a time it redirected you to Rune forum and then in rAudio -1 / discussion page (if I remember well)
Now it displays this:
<img src=https://user-images.githubusercontent.com/13005709/105997849-bc08b880-60b4-11eb-96e8-9c36b2101b1f.PNG width=300>

**Comment by AndyFx12** [comment]
Hi kosto-stavr,
top-left-corner-> it' s normal since some versions. Discussions-link can be found on help-page.

Did you refresh your library after mounting your USB-Stick?
Which filesystem are you using on this stick?

**Reply by kosto-stavr** [reply]
hi Andy,

- i didn't knew /notice it - thanks !
- Fat32 - I haven't refresh the library, its quite big and take some time - to tell the truth i always believed that auto mount was always followed by auto update.

thanks any way
SK

**Comment by kosto-stavr** [comment]
BTW 
where is this "help-page" ?

**Comment by AndyFx12** [comment]
Tap "?" for help-page:
![Screenshot 2021-01-27 164315](https://user-images.githubusercontent.com/35779271/106015584-ea8b9100-60be-11eb-9f8c-deba547bac9b.jpg)
Update of library with "only changed files" shouldn´t take very long. There´s an "auto-update"-option at MPD-settings. It´s worth a try.

**Comment by rern** [comment]
Fixed.
